shenyu-agent/shenyu-agent-plugins/shenyu-agent-plugin-metrics/shenyu-agent-plugin-metrics-prometheus/pom.xml

```diff
@@ -28,6 +28,11 @@
     <artifactId>shenyu-agent-plugin-metrics-prometheus</artifactId>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.shenyu</groupId>
+            <artifactId>shenyu-common</artifactId>
```

Comment:
why need add this jar？

Reply:
use dto in shenyu-common module

Reply:
and some utils class

Reply:
pls customize the agent module， not dependency shenyu-common，it too heavy~

Reply:
ok